syslog-ng-helm-chart/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "syslog-ng.fullname" . }}-test-connection"
+  labels:
+    {{- include "syslog-ng.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test-success
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
+      args: ['{{ include "syslog-ng.fullname" . }}:{{ .Values.service.port }}']
```

Comment:
Is there anything that listens on http within the container? I couldn't find anything.

Reply:
This is helm test. It test if a container can get (using the k8s dns) something on the service.
For this you need to understand how internal dns works. It has nothing to do with syslog-ng .
https://helm.sh/docs/topics/chart_tests/